services/motion/builtin/move_request.go

```diff
@@ -23,9 +23,21 @@ const (
 	defaultMaxReplans       = -1 // Values below zero will replan infinitely
 )
 
+// ValidatedMotionConfiguration is a copy of the motion.MotionConfiguration type
+// which has been validated to conform to the expectations of the builtin
+// motion servicl.
+type ValidatedMotionConfiguration struct {
```

Comment:
This should be private. Otherwise anything could create one and it will not be "Validated"

Reply:
done